PreMailer.Net/PreMailer.Net/Sources/DocumentStyleTagCssSource.cs

```diff
@@ -13,7 +13,7 @@ public DocumentStyleTagCssSource(IElement node)
 
 		public string GetCss()
 		{
-			return _node.InnerHtml;
+			return _node.FindChild<IText>()?.Data ?? "";
```

Comment:
The way `IText` is found and its Data accessed, is less obvious than `InnerHtml`. To offer the same level of intuitiveness, could it be moved out to an extension method or similar? That would also avoid duplication with code above, as both could use the method.

Such method should be allowed to return `null` and each point of invocation can fall back to their desired value.

Reply:
I added an extension method. The name is very literal and not specific to just CSS, feel free to change it as you like.

Test-wise I think the tests I added for the extension method specifically are enough. The existing tests already verify behavior when style elements are empty for example.